configs/katello/3.15.yaml

```diff
@@ -80,5 +80,5 @@
         katello-pulpcore-3.15-el7: 0
     build_package_group_source_tag: katello-pulpcore-nightly-el7-build
     external_repos:
-      - centos7-os
-      - centos7-updates
+      - centos-7-server
```

Comment:
Can we standardize on `centos-7-os` instead? That's the repo name that CentOS actually uses (as well as updates). There is no such thing as CentOS Server.

Reply:
I can do that as a follow up step, once everything is using a single repository, renames become easier.

Reply:
If anything, I think we should remove `centos-7-server` then. Note you are renaming updates as well here by adding a dash.

Reply:
Right now, I want to aim for alignment first (and removing duplication). Then do mass updates. The change to remove `centos-7-server` would at present touch 187 tags:

```
$ grep -r centos-7-server configs/ | wc -l
187
```

Now we wont actually need to update all of them but I want to give an indicator of my preferred strategy here.